pypeit/spectrographs/keck_deimos.py

```diff
@@ -741,8 +741,9 @@ def get_rawimage(self, raw_file, det):
         mosaic = None if nimg == 1 else self.get_mosaic_par(det, hdu=hdu)
         detectors = [self.get_detector_par(det, hdu=hdu)] if nimg == 1 else mosaic.detectors
 
-        if hdu[0].header['AMPMODE'] != 'SINGLE:B':
-            msgs.error('PypeIt can only reduce images with AMPMODE == SINGLE:B.')
+        # TODO check that that read noise and gain are the same for this amplifier mode??
```

Comment:
maybe remove this and leave for the forthcoming DEIMOS PR?

Reply:
Agreed.

Reply:
I leave this for @jhennawi